static/planttracer.js

```diff
@@ -585,7 +585,7 @@ function list_movies_data( movies ) {
 // The functions after this implement the interactivity
 //
 function list_ready_function() {
-    console.log("list_movies");
+    console.log("list_ready_function()");
```

Comment:
I think that the publish and unpublish buttons were form buttons previously which forced a form submit and a reload, and that you took out the reload elsewhere. This looks great.